src/psyclone/line_length.py

```diff
@@ -30,11 +30,9 @@ class FortLineLength(object):
     line wraps any lines that are larger than the specified line
     length'''
 
+    # pylint: disable=too-many-instance-attributes
     def __init__(self, line_length=132):
         self._line_length = line_length
-        self._line_types = ["statement", "openmp_directive",
```

Comment:
rm unused code?

Reply:
Not sure what you mean? Is this a question? Then yes, this attribute was not used.

Reply:
Sorry, yes that was a question!